src/helpers/translations/translations.js

```diff
@@ -1,20 +1,21 @@
 export default {
   locale: "en",
   crypto: {
+    bluetext: "Always for sale",
     ethereum: "ethereum based",
     tokens: "tokens",
     subHeading:
-      "Let your digital assets make a valuable contribution to the world.",
+      "Link your social profile to your preferred public address and verifiably showcase your patronage on our platform.",
     nft: "Non-Fungible Token",
     whatIsANFT:
       " is a way to ensure digital assets are unique and easily tradable on a blockchain.",
     harbergerTax: "Harberger Tax"
   },
   muggle: {
-    ethereum: "digital",
-    tokens: "animals",
-    subHeading:
-      "Become the guardian of a wildcard and contribute to endangered animals",
+    bluetext: "",
```

Comment:
```suggestion
    bluetext: "Always",
```

I wonder if this should be "Always raising funds for conservation", instead of "Raising funds for conservation"

Reply:
Also, the spacing was a bit funny here without something here. And now with the countup going every 2 seconds it is very clear what is meant by always (I hope).